src/lib_math.c

```diff
@@ -96,6 +96,51 @@ LJLIB_ASM_(math_max)		LJLIB_REC(math_minmax IR_MAX)
 
 LJLIB_PUSH(3.14159265358979323846) LJLIB_SET(pi)
 LJLIB_PUSH(1e310) LJLIB_SET(huge)
+LJLIB_PUSH(9007199254740992) LJLIB_SET(maxinteger)	/* 2^53 */
+LJLIB_PUSH(-9007199254740992) LJLIB_SET(mininteger)
```

Comment:
I suppose that's an unfortunate limitation of always storing numbers as float; lua is able to convert all the way up to `2^63-1` both ways when the input is an integer (i.e. `9223372036854775807` and not `9223372036854775807.0`), which is what it should be.  Please mention this difference in the documentation, hopefully we can fix this up in future.